### PR TITLE
Added player and obstacle images

### PR DESCRIPTION
**Pull request by renee-k**
Made one dino image for the player art. Made three obstacle images that are randomly selected to be displayed in gameplay.

### CODE REVIEW

File: Obstacle.py [hunk 1]
```diff
@@ -7,8 +7,27 @@
...
+            self.height = ROCK_HEIGHT
+            self.img = pg.image.load('assets/rock.png').convert_alpha()
+        # grass
+        else:
+            self.width = GRASS_WIDTH
+            self.height = GRASS_HEIGHT
+            self.img = pg.image.load('assets/grass.png').convert_alpha()
+
+        # self.width = random.randint(MIN_WIDTH, MAX_WIDTH)
+        # self.height = random.randint(MIN_HEIGHT, MAX_HEIGHT)
```
Comment (ggggg) [inline Obstacle.py:30]:
```suggestion
```

No need for this code

File: Obstacle.py [hunk 2]
```diff
@@ -18,7 +37,8 @@ def __init__(self, surface: pg.Surface):
         self.speed = random.randint(MIN_SPEED, MAX_SPEED)
 
     def show(self, surface: pg.Surface):
-        pg.draw.rect(surface, self.color, self.rect)
+        # pg.draw.rect(surface, self.color, self.rect)
```
Comment (ggggg) [inline Obstacle.py:40]:
```suggestion
```

File: Player.py [hunk 3]
```diff
@@ -12,12 +12,15 @@ def __init__(self, surface: pg.Surface):
...
                                  self.height)
+        self.img = pg.image.load('assets/dino.png').convert_alpha()
+
         self.jumping = False
         self.velocity = 0
         self.sound = Sound()
 
     def show(self, surface: pg.Surface):
-        pg.draw.rect(surface, PLAYER_COLOR, self.rect)
+        # pg.draw.rect(surface, PLAYER_COLOR, self.rect)
```
Comment (ggggg) [inline Player.py:22]:
```suggestion
```

File: constants.py [hunk 4]
```diff
@@ -26,3 +26,9 @@
 MAX_HEIGHT = 100
 MIN_SPEED = 400
 MAX_SPEED = 400
+TREES_WIDTH = 140
+TREES_HEIGHT = 160
+ROCK_WIDTH = 100
+ROCK_HEIGHT = 100
+GRASS_WIDTH = 160
+GRASS_HEIGHT = 90
```
Comment (ggggg) [inline constants.py:34]: Can we make these a bit smaller? the game is a bit hard to play at the moment

**Review by ggggg — CHANGES_REQUESTED** [review]
Thank you for contributing, this is a great PR!
Just a couple of small changes before we merge.

**Comment by renee-k** [comment]
Thanks for the feedback I believe the images should be small enough now!

**Comment by ggggg** [comment]
Can you make them even smaller? The game is not really playable right now with those sizes

**Review by ggggg — APPROVED** [review]

**Pull request merged**